app/services/json_validator.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class JsonValidator
+  attr_reader :schema, :data
+
+  def initialize(schema, data)
+    @schema = schema
+    @data = data.deep_stringify_keys
+  end
+
+  def invalid?
+    !validator.valid?(data)
+  end
```

Comment:
```suggestion
  def valid?
    validator.valid?(data)
  end
```

I find truthy statements are easier to read.